### PR TITLE
Add Shockbyte

### PR DESCRIPTION
**Pull request by TnrR**
**Added shockbyte.com.json**

- Domain: https://shockbyte.com
- Methods: TOTP Only
- Notes: The control panel and client area use separate accounts. So, 2FA will need to be enabled separately for each site
- Image: https://shockbyte.com/branding/logomarks/shockbyte_logomark.svg
- Docs: https://shockbyte.com/billing/knowledgebase/23/How-to-Enable-2-Factor-Authentication-in-Your-Shockbyte-Account.html
- Categories: Hosting, Gaming

### CODE REVIEW

File: entries/s/shockbyte.com.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  “Shockbyte”: {
+    "domain": “https://shockbyte.com”,
+    "img": "https://shockbyte.com/branding/logomarks/shockbyte_logomark.svg”,
```
Comment (hkamran80) [inline entries/s/shockbyte.com.json:4]: Please download the SVG and place it in the `img` directory

**Review by hkamran80 — CHANGES_REQUESTED** [review]
Please reformat the JSON using normal quotes, not smart quotes. Additionally, please separate the control panel and client entries if they use separate authentication systems.

File: entries/s/shockbyte.com.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    "tfa": [
+      “totp”
+    ],
+    "documentation": "https://shockbyte.com/billing/knowledgebase/23/How-to-Enable-2-Factor-Authentication-in-Your-Shockbyte-Account.html",
```
Comment (Carlgo11) [inline entries/s/shockbyte.com.json:8]:
```suggestion
    "documentation": "https://shockbyte.com/billing/knowledgebase/23",
```

File: entries/s/shockbyte.com.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
+{
+  “Shockbyte”: {
+    "domain": “https://shockbyte.com”,
```
Comment (Carlgo11) [inline entries/s/shockbyte.com.json:3]:
```suggestion
    "domain": "shockbyte.com",
```

File: entries/s/shockbyte.com.json [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+    "img": "https://shockbyte.com/branding/logomarks/shockbyte_logomark.svg”,
+    "tfa": [
+      “totp”
+    ],
+    "documentation": "https://shockbyte.com/billing/knowledgebase/23/How-to-Enable-2-Factor-Authentication-in-Your-Shockbyte-Account.html",
+	“notes”: “The control panel and client area use separate accounts. So, 2FA will need to be enabled separately for each site.”
+    "categories": [
+      “Hosting”,
+      “Gaming”
```
Comment (Carlgo11) [inline entries/s/shockbyte.com.json:12]:
```suggestion
      "hosting"
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

**Comment by Carlgo11** [comment]
As the requested changes haven't been resolved yet I'm going to label this PR as inactive.
> **Warning**
> Unless there's activity within the next 7 days I'll close the PR.

**Comment by Carlgo11** [comment]
Closing due to inactivity.
> **Note**
> The PR author you can still commit the requested changes and re-open this PR.
> If someone else wants to commit the requested changes you can create a new PR and link to this PR.

**Pull request closed without merging**